src/explore-education-statistics-common/src/modules/data-catalogue/components/ApiDataSetQuickStart.tsx

```diff
@@ -116,14 +114,20 @@ export default function ApiDataSetQuickStart({
         labelHidden={false}
         url={`${publicApiVersionUrl}/data-sets/${dataSetId}/query?dataSetVersion=${dataSetVersion}`}
       />
-      <p>
+      <p className="govuk-!-margin-bottom-8">
         {renderLink({
           to: `${publicApiDocsVersionUrl}/endpoints/QueryDataSetGet/`,
           children: 'Guidance: Query data set (GET)',
         })}
       </p>
 
-      <Heading>Query data set (POST)</Heading>
+      <Heading>Query data set using POST</Heading>
+      <p>
+        The POST endpoint is recommended for production level pipeline queries.
+        POST requires a query to be built and supplied in json format using
```

Comment:
Do you know if `json` should be lowercased?  I'm not sure that the correct convention is in general text.

Reply:
Yeah it should be uppercase, I've changed it now.